connector/countconnector/config.go

```diff
@@ -47,8 +47,10 @@ type MetricInfo struct {
 }
 
 type AttributeConfig struct {
-	Key          string `mapstructure:"key"`
-	DefaultValue string `mapstructure:"default_value"`
+	Key               string  `mapstructure:"key"`
+	DefaultValue      string  `mapstructure:"default_value"`
+	DefaultIntValue   int64   `mapstructure:"default_int_value"`
+	DefaultFloatValue float64 `mapstructure:"default_float_value"`
```

Comment:
I think this design introduces unnecessary relationships between fields in the config. If we cannot use a type constraint, can we just make `DefaultValue` an `any`?